CONTRIBUTING.md

```diff
@@ -104,48 +93,34 @@ conda uninstall pytorch -y
 yes | pip uninstall torch
 ```
 
-2. Clone a copy of PyTorch from source:
-
-```bash
-git clone https://github.com/pytorch/pytorch
-cd pytorch
-```
+* If you want to have no-op incremental rebuilds (which are fast), see [Make no-op build fast](#make-no-op-build-fast) below.
 
-If you already have PyTorch from source, update it:
+* Instead of installing PyTorch via `python setup.py install`, use `python setup.py develop`.
 
-```bash
-git pull --rebase
-git submodule sync --recursive
-git submodule update --init --recursive
-```
+  This mode will symlink the Python files from the current local source
+  tree into the Python install.  This way when you modify a Python file, you
+  won't need to reinstall PyTorch again and again.  This is especially
+  useful if you are only changing Python files.
```

Comment:
This explanation might make more sense in the README.md file since that's where the `develop` installation mode is specified

Reply:
Ack